internal/xds/rbac/rbac_engine.go

```diff
@@ -285,16 +309,16 @@ type rpcData struct {
 	certs []*x509.Certificate
 }
 
-func (engine *engine) doAuditLogging(rpcData *rpcData, rule string, authorized bool) {
+func (e *engine) doAuditLogging(rpcData *rpcData, rule string, authorized bool) {
 	event := &audit.Event{
 		FullMethodName: rpcData.fullMethod,
 		Principal:      rpcData.peerInfo.Addr.String(),
```

Comment:
Will this be the Spiffe ID from the cert?

Reply:
Changed to support Spiffe IDs since this is the world of RBAC